presto-docs/src/main/sphinx/release/release-0.248.rst

```diff
@@ -0,0 +1,37 @@
+=============
+Release 0.248
+=============
+
+**Highlights**
+==============
+* New aggregation function MAP_UNION_SUM.
+* New scalar function DESTRUCTURE_TDIGEST.
+* Support for session property managers, that can override session properties set by a user.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve output buffer memory tracking to reduce lock contention.
+* Add support for overriding session properties using session property managers :doc:`/admin/session-property-managers`. Setting `overrideSessionProperties` to true will cause the property to be overridden and remain overridden even if subsequent rules match the property but don't have `overrideSessionProperties` set.
+* Add support to drop multiple UDFs at the same time.
+* Add a REST endpoint to get TaskInfo, without directly going to worker endpoint.
+* Add new aggregation function :func:`map_union_sum`.
+* Add new scalar function DESTRUCTURE_TDIGEST to expose the internal TDigest data structure as a ROW
+* Add support to configure ZSTD compression level for ORC writer.
+* Add warning for JOIN conditions with OR expressions.
+* Add configuration property `internal-communication.https.trust-store-password` to set the Java Truststore password used for https in internal communications between nodes.
+
+Hive Connector Changes
+________________
+* Add session property `temporary_table_create_empty_bucket_files` and configuration property `hive.create-empty-bucket-files-for-temporary-table`, which, when set to false, disables the creation of zero-row files for temporary table empty buckets, to improve performance.
```

Comment:
the keywords should be in double angle quotes

Reply:
done